supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs of Red Hat products. These references can confuse users and cause legal concerns. This information is best left to marketing materials.
+
+.Example phrases
+
+* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to me the product is free when it isn't.
```

Comment:
Would it be helpful to add an example about avoiding "free" as well as "cost"? Optionally link to IBM SG section on "free" as in original discussion.

Reply:
Do we need to scope cost a bit more? Do we need to state monetary costs? Learning a product might cost time and resources.

Reply:
What about "no cost" or "without charge". These two phrases are mentioned in the IBM SG, but should also be misleading?

Should we state that an IBM SG entry exists for "free", but the RH SSG supersedes the guidance on the use of this term and related terms?

Reply:
Sounds good, @EricPonvelle . Thanks!